src/components/SearchAutocomplete/index.js

```diff
@@ -181,27 +201,27 @@ export default class SearchAutocomplete extends Component {
 
   renderList(key) {
     return this.state[key].map((item) => (
-      <li key={item.href} tabIndex="0" onKeyDown={((event) => { this.handleItemKeyDown.call(this, event, item); }).bind(this)}>
+      <li key={item.href} tabIndex="0" onKeyDown={(event) => this.handleItemKeyDown(event, item)}>
         <div className={styles.link}>
           <a href={item.href} tabIndex="-1">{item.href}</a>
         </div>
         <div className={styles.text}>
-          <a href={item.href} tabIndex="-1" dangerouslySetInnerHTML={{__html: item.text }} />
+          <a href={item.href} tabIndex="-1" dangerouslySetInnerHTML={{__html: item.text}} />
         </div>
       </li>
     ));
-  };
+  }
 
   render() {
     const { surahs, ayat } = this.state;
 
     return (
       <div className={`${styles.autocomplete} ${ayat.length || surahs.length ? '' : 'hidden'}`}>
```

Comment:
`index.js?8fa9*:219 Uncaught TypeError: Cannot read property 'length' of undefined`

this happened upon initial click/focus into the search field